cli/args/flags.rs

```diff
@@ -4956,7 +4954,18 @@ fn permission_args_parse(flags: &mut Flags, matches: &mut ArgMatches) {
   }
 
   if let Some(env_wl) = matches.remove_many::<String>("allow-env") {
-    flags.permissions.allow_env = Some(env_wl.collect());
+    let env_permissions: Vec<String> = env_wl.collect();
+    for env_var in &env_permissions {
+      if env_var.contains('*') {
```

Comment:
`--allow-env=*` makes no sense because it's equal to `--allow-env`. If we are to support "wildcards" in env var names it should only be at the begging or end (`AWS_*`, `*_DENO`)

Reply:
OK I will remove it

Reply:
> `--allow-env=*` makes no sense because it's equal to `--allow-env`. If we are to support "wildcards" in env var names it should only be at the begging or end (`AWS_*`, `*_DENO`)

done